src/bsoncxx/types/value.cpp

```diff
@@ -1,3 +1,4 @@
+// audit
```

Comment:
This was just so we could see the .cpp file in the review, to facilitate an audit of the code.